src/main/java/Models/Training.java

```diff
@@ -0,0 +1,13 @@
+package Models;
+
+import java.time.LocalDateTime;
+
+public interface Training {
```

Comment:
interfaces are used more for defining the common behavior of implementing classes. in this case an abstract class would be more suitable